marklogic-data-hub/src/main/java/com/marklogic/hub/util/BeanUtil.java

```diff
@@ -0,0 +1,19 @@
+package com.marklogic.hub.util;
+
+
+import org.springframework.beans.BeansException;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ApplicationContextAware;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BeanUtil implements ApplicationContextAware {
```

Comment:
While Spring's method is called "getBean", I think a better name for this is "ApplicationContextReference". I'd also just put a "getApplicationContext" method on this so a client can do what it needs with the Spring context. "BeanUtil" sounds like a class that performs operations on JavaBeans.

Reply:
Fixed in 69488d8bda278aa2ab8b428bab6f587ad18f8b89